assets/js/src/segments/subscribers_counter.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import MailPoet from 'mailpoet';
+
+type SubscriberCount = {
```

Comment:
In the calculator, you were using a different syntax for the type. You used an interface and number | undefined. I think we should start unifying it. So instead of using {} here, I would recommend using `interface`. and instead of  using number | undefined I would use `?` syntax